outcomes/firefox_ios/default_browser.toml

```diff
@@ -0,0 +1,37 @@
+friendly_name = "Default browser"
+description = "Metrics that describe whether Firefox for iOS is a user's default browser."
+
+[metrics.default_browser_action]
+friendly_name = "Clicked to set Firefox as default"
+description = """
+    Fraction of users who clicked on a UI element to set Firefox as default
+    during the analysis window.
+    This only opens a system UI that allows users to change their default browser,
+    so not all of these clients will actually change their default browser.
+"""
+select_expression = "COUNTIF(event.name = 'default_browser_changed') > 0"
+data_source = "events"
+statistics = { binomial = {} }
+
+
+[metrics.opened_as_default]
+friendly_name = "Opened as default browser"
+description = """
+    Fraction of users who opened Firefox as default browser. 
+"""
+select_expression = "SUM(metrics.counter.app_opened_as_default_browser) > 0"
```

Comment:
I'm not sure how we want to compute this metric. The document says something about 7 day retention which is currently not reflected here.

Reply:
Can we update the description for "Opened as default browser" to be "Fraction of users who opened Firefox through an external link, implying the client has Firefox set as the default browser"?

Reply:
Updated the description